integration-tests/scripts/buildEvmClientTestMatrixList.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```

Comment:
Would it make sense to just update the buildTestMatrixList.sh script to handle this case instead of duplicating most of its code?

Reply:
@tateexon created a ticker for myself to tackle it in another PR